indico/modules/events/tracks/templates/management.html

```diff
@@ -20,15 +21,17 @@
         data-reload-after>
         {% trans %}Add track{% endtrans %}
     </a>
-        {% if not event.tracks %}
-            {% call message_box('info', classes='no-rules-placeholder') %}
-                {% trans %}No tracks have been added yet.{% endtrans %}
-            {% endcall %}
-        {% else %}
-            <ul id="track-list" class="i-box-group vert" data-url="{{ url_for('.sort_tracks', event) }}">
-                {{ render_track_list(event) }}
-            </ul>
-        {% endif %}
+    {% if not event.tracks %}
```

Comment:
the if/else part has to go inside the macro. otherwise it won't work when you delete the last / add the first track.